at_client/lib/src/service/notification_service_impl.dart

```diff
@@ -260,4 +259,47 @@ class NotificationServiceImpl
         '${_atClient.getCurrentAtSign()} monitor status: ${_monitor!.getStatus()}');
     return _monitor!.getStatus();
   }
+
+  @override
+  Future<NotificationResult> getStatus(String notificationId) async {
+    var status = await _atClient.notifyStatus(notificationId);
+    var atResponse = DefaultResponseParser().parse(status);
+    NotificationResult notificationResult;
+    // If the Notification Response is error, set the notification status to undelivered
+    if (atResponse.isError) {
+      return NotificationResult()
+        ..notificationID = notificationId
+        ..notificationStatusEnum = NotificationStatusEnum.undelivered
+        ..atClientException = AtClientException(
+            atResponse.errorCode, atResponse.errorDescription);
+    }
+
+    notificationResult = NotificationResult()
+      ..notificationID = notificationId
+      ..notificationStatusEnum =
+          _getNotificationStatusEnum(atResponse.response);
+    return notificationResult;
+  }
+
+  /// Returns the NotificationStatusEnum for the given string of notificationStatus
+  NotificationStatusEnum _getNotificationStatusEnum(String notificationStatus) {
+    switch (notificationStatus.toLowerCase()) {
+      case 'delivered':
+        return NotificationStatusEnum.delivered;
+      case 'undelivered':
+        return NotificationStatusEnum.undelivered;
+      case 'acks':
+        return NotificationStatusEnum.ackS;
```

Comment:
we can rename to NotificationStatusEnum.ackSent and NotificationStatusEnum.ackReceived

Reply:
Removed the status for now.